pages/governance/proposing_a_new_market.md

```diff
@@ -55,7 +55,46 @@ The following decribes how to set various parameters for a new market and assume
 | `Msg[Update/Create]ClobPair` | `step_base_quantums` | (aka step size): min increment in the size of orders (number of coins) on the CLOB in base quantums. | `1000000` |
 | `MsgDelayMessage` | `delay_blocks` | number of blocks before which the `MsgUpdateClobPair` is executed and transitions the market to `ACTIVE` | `3600` (equal to an hour at `1 sec` blocktime) |
 
-### exchange_config_json
+### Determining liquidity tier
+
+Liquidity tier is used to determine various risk and market parameters. A new market should be 2: Long-tail unless there are strong reasons to think that it is safe, in which case it can 1: Mid-cap. Liquidity tier and associated parameters can be updated through governance vote at a later date. A market may be considered for 1: Mid-cap if there are at least 8 robust spot markets with reasonable liquidity and 30d daily trading volume is at least $100M across exchanges.
```

Comment:
Can we merge this section to above around line 38, to avoid scattered information?

Reply:
sounds good! qq: what does `so mappings could change if these tiers change` mean? Is it that the formula from liquidity tier to parameters can change later?